bc_gym_planning_env/envs/base/action.py

```diff
@@ -2,11 +2,29 @@
 from __future__ import division
 from __future__ import absolute_import
 
-import attr
-import numpy as np
 
-
-@attr.s
 class Action(object):
     """ Object representing an 'action' - a motion primitive to execute in the environment """
-    command = attr.ib(type=np.ndarray)
+
+    def __init__(self, command=None):
```

Comment:
Please do not remove compatibility of others while introducing new functionality.

Reply:
refactored as a new class function